esi_ui/static/dashboard/esi/lessee/offers/esi-offers-table.controller.js

```diff
@@ -69,6 +70,15 @@
         });
       });
     }
+
+    function getStatusColor(availabilityStart) {
+      const currentTime = new Date();  // Current UTC time
+      const availabilityDateTime = new Date(availabilityStart.replace(" ", "T") + "Z");
+    
+      const timeDifference = (availabilityDateTime - currentTime) / 60000;
+    
+      return Math.abs(timeDifference) <= 5 ? 'green' : 'brown';
```

Comment:
I was thinking about this logic - what happens if you replace this with `<= 0`? I think it should still work.

Reply:
When `timeDifference <= 0,` it implies that the availabilityStart time has either already passed or is exactly the current time.
This approach will turn the color green for any offer that is available at the current time or was available in the past.
Future Time:
If `timeDifference > 0 `(availability is in the future), the color will be brown.

Reply:
Yep - I'm asking if you can just change it to immediate availability, which I believe `<= 0` should do.

Reply:
done.